make/prepare

```diff
@@ -275,6 +275,10 @@ if rcp.has_option("configuration", "redis_url"):
 else:
     redis_url = ""
 
+if rcp.has_option("configuration", "skip_reload_env_reg"):
+    skip_reload_env_reg = rcp.get("configuration", "skip_reload_env_reg")
```

Comment:
Is calling the attribute `reg` a convention?  I thought normally it should be called as "pattern"?

Reply:
Updated